src/Humanizer.Tests/TimeSpanHumanizeTests.cs

```diff
@@ -82,8 +82,8 @@ public void TimeSpanWithMaxTimeUnit(int ms, string expected, TimeUnit maxUnit)
         }
 
         [Theory]
-        [InlineData(0, 3, "no time")]
```

Comment:
Yes please revert these changes. We can open separate issue for "no time"/"just now" and discuss there.